custom.config

```diff
@@ -0,0 +1,271 @@
+/*
```

Comment:
The `custom.config` file isn't supposed to be pushed to git since the only reason for its existence is the user-specific modifications which you need for your machine.